ftplugin/elixir.vim

```diff
@@ -18,3 +18,6 @@ endif
 
 setlocal comments=:#
 setlocal commentstring=#\ %s
+
+setlocal iskeyword+=.
```

Comment:
I don't know if I agree with this, since this will change the autocomplete behaviour too.